### DISCUSSION TITLE
Duda pedidos contaminados

### DISCUSSION THREAD

**Discussion by mpalaciosc** [IIC2133-PUC/2024-1#12] [Tarea 0] [answered]
Holaa!
Acabo de leer el enunciado y no me quedo muy claro cuando nosotros podemos identificar que un pedido esta contaminado.
Se supone que es un evento que indica el id del pedido, o como seria?

**Comment by sachondo7** [comment, marked as answer]
Sip, si te fijas el comando recibe como input tanto el id de la nave como el id del pedido que está contaminado, y a partir de eso tienes que buscar si existe ese pedido y eliminarlo

**Comment by mpalaciosc** [comment]
Aa perfecto! no habia entendido muy bien el enunciado jaja, graciass!
